samples/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.7
```

Comment:
1) rename directory to `docker` or `build` so that users are not confused about the contents.  `samples` sounds like it contains data.
2) rename script from `sample` to `examples`.  I think this is better naming and is consistent with our naming of the image.